From ff463b7cdfa7a5364778ea965dcaa0e78d1cfb9d Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <jkysela@redhat.com>
Date: Wed, 3 Sep 2014 07:17:38 -0400
Subject: [alsa] hda: Rename ASUS subwoofer quirks

Message-id: <140972823573-90-patchset-jkysela@redhat.com>
Patchwork-id: 90790
O-Subject: [RHEL 7.1 PATCH 089/442] ALSA: hda - Rename ASUS subwoofer quirks
Bugzilla: 1112200
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>

Just a small refactoring to make the next patch slightly simpler.

Signed-off-by: David Henningsson <david.henningsson@canonical.com>
Signed-off-by: Takashi Iwai <tiwai@suse.de>

(picked from kernel: 8e54b4accdf0cc30a0246fdc4cced0c2f463254a)
rhbz#1112200
Signed-off-by: Jaroslav Kysela <jkysela@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/sound/pci/hda/patch_realtek.c b/sound/pci/hda/patch_realtek.c
index 0d49205..7e59a7e 100644
--- a/sound/pci/hda/patch_realtek.c
+++ b/sound/pci/hda/patch_realtek.c
@@ -4926,9 +4926,9 @@ enum {
  ALC662_FIXUP_INV_DMIC,
  ALC668_FIXUP_DELL_MIC_NO_PRESENCE,
  ALC668_FIXUP_HEADSET_MODE,
- ALC662_FIXUP_BASS_CHMAP,
+ ALC662_FIXUP_BASS_MODE4_CHMAP,
  ALC662_FIXUP_BASS_1A,
- ALC662_FIXUP_BASS_1A_CHMAP,
+ ALC662_FIXUP_BASS_CHMAP,
  ALC668_FIXUP_AUTO_MUTE,
 };
 
@@ -5116,7 +5116,7 @@ static const struct hda_fixup alc662_fixups[] = {
   .type = HDA_FIXUP_FUNC,
   .v.func = alc_fixup_headset_mode_alc668,
  },
- [ALC662_FIXUP_BASS_CHMAP] = {
+ [ALC662_FIXUP_BASS_MODE4_CHMAP] = {
   .type = HDA_FIXUP_FUNC,
   .v.func = alc_fixup_bass_chmap,
   .chained = true,
@@ -5128,12 +5128,12 @@ static const struct hda_fixup alc662_fixups[] = {
    {0x1a, 0x80106111}, /* bass speaker */
    {}
   },
+  .chained = true,
+  .chain_id = ALC662_FIXUP_BASS_CHMAP,
  },
- [ALC662_FIXUP_BASS_1A_CHMAP] = {
+ [ALC662_FIXUP_BASS_CHMAP] = {
   .type = HDA_FIXUP_FUNC,
   .v.func = alc_fixup_bass_chmap,
-  .chained = true,
-  .chain_id = ALC662_FIXUP_BASS_1A,
  },
 };
 
@@ -5155,9 +5155,9 @@ static const struct snd_pci_quirk alc662_fixup_tbl[] = {
  SND_PCI_QUIRK(0x1028, 0x0628, "Dell", ALC668_FIXUP_AUTO_MUTE),
  SND_PCI_QUIRK(0x1028, 0x064e, "Dell", ALC668_FIXUP_DELL_MIC_NO_PRESENCE),
  SND_PCI_QUIRK(0x103c, 0x1632, "HP RP5800", ALC662_FIXUP_HP_RP5800),
- SND_PCI_QUIRK(0x1043, 0x11cd, "Asus N550", ALC662_FIXUP_BASS_1A_CHMAP),
- SND_PCI_QUIRK(0x1043, 0x1477, "ASUS N56VZ", ALC662_FIXUP_BASS_CHMAP),
- SND_PCI_QUIRK(0x1043, 0x1bf3, "ASUS N76VZ", ALC662_FIXUP_BASS_CHMAP),
+ SND_PCI_QUIRK(0x1043, 0x11cd, "Asus N550", ALC662_FIXUP_BASS_1A),
+ SND_PCI_QUIRK(0x1043, 0x1477, "ASUS N56VZ", ALC662_FIXUP_BASS_MODE4_CHMAP),
+ SND_PCI_QUIRK(0x1043, 0x1bf3, "ASUS N76VZ", ALC662_FIXUP_BASS_MODE4_CHMAP),
  SND_PCI_QUIRK(0x1043, 0x8469, "ASUS mobo", ALC662_FIXUP_NO_JACK_DETECT),
  SND_PCI_QUIRK(0x105b, 0x0cd6, "Foxconn", ALC662_FIXUP_ASUS_MODE2),
  SND_PCI_QUIRK(0x144d, 0xc051, "Samsung R720", ALC662_FIXUP_IDEAPAD),
-- 
1.7.1